webapp/src/components/task_creation_form/TaskForm.vue

```diff
@@ -116,6 +128,16 @@
           </div>
         </IconCard>
       </div>
+      <IconCard>
+        <template #title> How to join after ?</template>
+        <div>
+          <p>
+            After submitting the form, others will be able to join the task from the
+            <span @click="goToDiscollaborative()" class="underline font-bold cursor-pointer">DISCOllaborative page</span>. You can find more explanations about
+            Disco in the <span @click="goToInformation()" class="underline font-bold cursor-pointer">Information page</span>.
+          </p>
```

Comment:
We actually have components for DISCOllaboratives and DISCO! We also have been displaying hyperlinks with `text-blue-400` (we should have created a component for this already...)
```suggestion
            <span @click="goToDiscollaborative()" class="underline text-blue-400 cursor-pointer"><DISCOllaboratives /> page</span>. You can find more explanations about
            <DISCO /> in the <span @click="goToInformation()" class="underline text-blue-400 cursor-pointer">Information page</span>.
```
Make sure to import the components:
```
import DISCO from "@/components/simple/DISCO.vue";
import DISCOllaboratives from "@/components/simple/DISCOllaboratives.vue";
```